com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/Configuration.java

```diff
@@ -147,7 +147,7 @@ public static Configuration getInstance() {
     public static final Integer DEFAULT_CHANGE_LOG_LOCK_WAIT_TIME = 1; // 1 minute(s)
     public static final Integer DEFAULT_CHANGE_LOG_LOCK_DURATION = 1; // 1 minute(s)
     public static final Integer DEFAULT_CHANGE_LOG_LOCK_ATTEMPTS = 5; // 5 minute(s)
-    public static final Boolean DEFAULT_GATHER_USAGE_STATISTICS = false;
+    public static final Boolean DEFAULT_GATHER_USAGE_STATISTICS = true;

```

Comment:
For XSA on-premise, we should not gather usage statistics by default. It would be better to keep the default to **false** and an **true** configuration in the manifest for CF.